strider/fetcher.py

```diff
@@ -164,6 +164,9 @@ async def generate_from_kp(
             )
         for batch_results in batch(onehop_results, 1_000_000):
             result_map = defaultdict(list)
+            # copy subqgraph between each batch
+            # before we fill it with result curies
+            populated_subqgraph = copy.deepcopy(subqgraph)
```

Comment:
I think another comment or two describing why we need to do this would be helpful. Just describe the problem we are solving right now.